.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "presets": ["react", "env", "stage-1"],
-  "env": {
-    "development": {
-      "presets": ["react-hmre"]
-    }
-  }
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
@jbuck94 when I tried to run build:client I'm getting an error "Couldn't find preset “es2015” relative to directory". I think that you will need to install babel-preset-es2015 via npm or add the presets for babel on webpack.config

Reply:
Adding my two cents here, but maybe you can seize the chance to change babel's preset to `babel-preset-latest` since the syntax has changed on latest releases.

https://babeljs.io/docs/plugins/preset-latest/